news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/blanche.jpg)](https://osu.ppy.sh/beatmapsets/759524)
+
+Dormin's "Une mage blanche" shows [Sulfur](https://osu.ppy.sh/users/5297447)'s expertise of movement and flow throughout the Insane diff very well and makes this map a blast to play alongside neat usage of default sample hitsounds, giving the map a nice classic feel. However, don't overestimate this map as it will test your finger control with its frequent use of triples and 3/4 sliders combined with circle heavy sections. If you're still looking for a bigger challenge, go ahead and try out [Kalibe](https://osu.ppy.sh/users/3376777)'s awesome Extra which will challenge your aim even harder while keeping that same effortless movement through Russian inspired patterns!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/good-night.jpg)](https://osu.ppy.sh/beatmapsets/679935)
+
+[Chaoslitz](https://osu.ppy.sh/users/3621552) and Moecho’s collab on colate and Nanahira’s "Good Night" is a great example of how collab difficulties are supposed to be mapped: the styles of both mappers blending with each other, resulting in a smooth playing experience without either of the mappers’ parts outshining the other. Coupled with the intuitive rhythm and patterns, the result is a very enjoyable map to play for players of all skillsets and levels.
+
+
+## <a name="taiko" id="taiko"></a>osu!taiko
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/passage.jpg)](http://osu.ppy.sh/beatmapsets/587426)
+
+The winning mapset of this season's beatmap Spotlight is nothing less than The Flashbulb's "Passage D", mapped by [Raiden](https://osu.ppy.sh/users/2239480)!
+
+A song known across several games and game modes, it has found its way into osu!taiko's ranked section a few months after being featured as a map in the 2018 osu!taiko World Cup Finals. Coincidence? Who knows. What I do know is that this entire map set is incredibly well-mapped, which is a big accomplishment given the calm yet chaotic nature of this song. 
+
+Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances playability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well. And yet, it follows the song very nicely, its flow (and "anti-flow") works, and that extends to all difficulties.
```

Comment:
"It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping."

what? how would you find different snaps if it's specifically in a chain of 1/4? this is a bit confusing the way it's currently written.

Reply:
Even "It's not uncommon to find 4 different snaps mixed in with a chain of 1/4 notes, the common snapping." would be better